### PR TITLE
Feat/add reviews related movies

### PR DESCRIPTION
**Pull request by arjun0419**
https://trello.com/c/bECxBySh/31-set-up-proxy-server-and-render-all-modules-using-the-proxy-server

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+.DS_Store
+client/dist/bundle.js
```
Comment (lee-philip-n) [inline .gitignore]: there is no bundle here

File: proxy/.gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+node_modules
```
Comment (lee-philip-n) [inline proxy/.gitignore]: why is there two gitignore files?

File: proxy/server/app.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+const routes = require('./routes.js')
+
+app.use(morgan(':method :url :response-time'))
+
+const statFilesPath = path.resolve(__dirname, '../client/dist');
+
+app.use(express.static(statFilesPath));
+app.use('/movies', routes);
+
+// app.get('/', (req, res) => {
```
Comment (lee-philip-n) [inline proxy/server/app.js]: comments need to be removed if not used

File: proxy/.gitignore (hunk 2 above)
Comment (arjun0419) [reply proxy/.gitignore]: deleted

File: proxy/server/app.js (hunk 3 above)
Comment (arjun0419) [reply proxy/server/app.js]: done

**Pull request merged**